examples/sdk/reactNative/package.json

```diff
@@ -10,25 +10,22 @@
     },
     "dependencies": {
         "@backtrace/react-native": "file:../../../packages/react-native",
-        "react": "18.2.0",
-        "react-native": "0.72.4"
+        "@backtrace/react": "file:../../../packages/react",
```

Comment:
why do you need react ?

Reply:
uh, not sure, I thought it was required. Removed.